e2e/libs/k8s/k8s.py

```
@@ -170,3 +171,25 @@ def delete_namespace(namespace):
         api.delete_namespace(name=namespace)
     except ApiException as e:
         assert e.status == 404
+
+def wait_for_namespace_pods_running(namespace):    
+    retry_count, retry_interval = get_retry_count_and_interval()
+
+    for i in range(retry_count):        
```

Ruff

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)